elifearticle/article.py

```diff
@@ -22,6 +26,23 @@ def __unicode__(self):
                 unicode_dict[key] = unicode(self.__dict__.get(key))
         return unicode(unicode_dict)
 
+    def __str__(self):
+        """
+        Return `str` representation of the simple object properties,
+        if there is a list or dict just return an empty representation
+        for easier viewing and test case scenario writing
+        """
+        _dict = {}
+        for key in self.__dict__:
+            if isinstance(self.__dict__.get(key), list):
+                _dict[key] = []
+            elif isinstance(self.__dict__.get(key), dict):
```

Comment:
is `OrderedDict` used around here and/or falling into this conditional?

Reply:
No its not and I think its right to not try and imply order at this level, the issue seemed to stem even from retrieving nested values (old code thats now been fixed):
`self.assertEqual(unicode_value(article_object.dates.get('pub')), "{'date_type': u'pub', ....some values}`

Reply:
There were some fixture file comparisons in tests before, but due to the problems with converting to unicode I deleted them. I only had them there to speed up development until I was able to create individual tests and then the full file comparisons were no longer required.

The string values of object I would hope would only be used in tests, and the Article ``pretty()`` function has no real purpose (I think) unless someone thinks of one in the future.